crates/muvm/src/net.rs

```diff
@@ -41,6 +41,8 @@ pub fn start_passt(server_port: u32) -> Result<UnixStream> {
     let child = Command::new("passt")
         .args(["-q", "-f", "-t"])
         .arg(format!("{server_port}:{server_port}"))
+        .arg("-t")
+        .arg(format!("{root_server_port}:{root_server_port}"))
```

Comment:
Should be combine with the above `-t`:

From the `passt` manpage:

> A comma-separated list of ports

Reply:
Done.